blogReader.py

```diff
@@ -16,18 +16,20 @@ def __init__(self) -> None:
         self.__new = []
         self.__getArticles()
         self.__cacheArticlesID()
-        self.handler = logging.getLogger(__name__)
+        self.logger = logging.getLogger(__name__)
 
     def __getArticles(self):
         if self.__cache == []:
             self.__cache = requests.get(self.url).json()
             return
         self.__new = requests.get(self.url).json()
+        self.logger.info("'getArticles' request sent")
```

Comment:
you should log the url you're requesting :wink: 
accordign to logging doc the format is:
`self.logger.info("'getArticles' request %s", <var_name>)`

Reply:
fixed by ba66297c86219c5589cd56f986245496c6e909e1